tsconfig.json

```diff
@@ -1,11 +1,10 @@
 {
   "compilerOptions": {
-    "outDir": "intermediate",
-    "allowJs": true,
+    "outDir": "runtime/ts-build",
+    "allowJs": false,
     "target": "es6"
   },
   "include": [
-    "runtime/**/*",
-    "platform/**/*"
-  ]
+    "runtime/ts/**/*",
```

Comment:
Can you keep the typescript files in the same directory and use runtime/**/*.ts instead?

Reply:
We could, but it gets tricky in terms of .gitignore. I don't want us to accidentally check in the .js build artifacts.